pkgs/development/tools/jq/default.nix

```
@@ -71,11 +74,11 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "A lightweight and flexible command-line JSON processor";
-    homepage = "https://stedolan.github.io/jq/";
+    homepage = "https://jqlang.github.io/jq/";
     license = licenses.mit;
-    maintainers = with maintainers; [ raskin globin artturin ];
+    maintainers = with maintainers; [ raskin globin artturin ncfavier ];
```

```suggestion
    maintainers = with maintainers; [ raskin artturin ncfavier ];
```
I would suggest to remove the least active maintainer to not have diffusion of responsibility